pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -94,6 +94,7 @@ type LoadBalancerOpts struct {
 type BlockStorageOpts struct {
 	BSVersion       string `gcfg:"bs-version"`        // overrides autodetection. v1 or v2. Defaults to auto
 	TrustDevicePath bool   `gcfg:"trust-device-path"` // See Issue #33128
+	AddLabel        bool   `gcfg:"add-volume-label"`
```

Comment:
‘add-volume-label’ is not good, how about 'ignore-volume-az'?
'AddLabel' --> 'IgnoreVolumeAZ'